pkg/endpoints/dashboard.go

```diff
@@ -186,3 +186,118 @@ func getPipelineDeployments(r Resource, namespace string) string {
 
 	return version
 }
+
+// Get triggers version
+func GetTriggersVersion(r Resource, isOpenShift bool) string {
```

Comment:
Could we reduce duplicate code in this file? GetDeploymentVersion perhaps and you accept triggers or pipelines, then just change the version = line from getTriggersDeployments if it's Triggers, getPipelineDeployments if it's Pipelines?

Reply:
I've reduced the duplicate code now 
Combined the TriggersDeployments and GetDeployments and version stuff